src/fights.ts

```diff
@@ -672,7 +688,7 @@ const freeFightSources = [
     },
     false,
     {
-      canOverrideMacro: true,
+      macroAllowsFamiliarActions: true,
```

Comment:
We don't really need this if it already defaults to true
(Not a major issue, just a thing about standarization - the other FreeFights don't have this)